tests/acceptance/features/apiSpaces/copySpaces.feature

```diff
@@ -187,17 +187,17 @@ Feature: copy file
     Given the administrator has given "Brian" the role "Space Admin" using the settings api
     And user "Brian" has created folder "/testshare"
     And user "Brian" has uploaded file with content "testshare content" to "/testshare/testshare.txt"
-    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "<permissions>"
+    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "<share-permission>"
     And user "Alice" has accepted share "/testshare" offered by user "Brian"
     When user "Alice" copies file "/testshare/testshare.txt" from space "Shares Jail" to "testshare.txt" inside space "Personal" using the WebDAV API
     Then the HTTP status code should be "201"
     And for user "Alice" the space "Personal" should contain these entries:
       | /testshare.txt |
     And for user "Alice" the content of the file "/testshare.txt" of the space "Personal" should be "testshare content"
     Examples:
-      | permissions |
-      | 31          |
-      | 17          |
+      | share-permission |
```

Comment:
IMO it's better to change the new ones to <permission>, it'll give you few changes